src/main.html

```diff
@@ -148,6 +159,7 @@
             </div>
           </div>
           <div class="content__cpu">
+            <h3 class="content__title">CPU</h3>
```

Comment:
Lets make this title not disappear on scroll.
It probably should be in separate container for that..